soccer/planning/tests/PlannerTest.cpp

```diff
@@ -68,11 +77,11 @@ TEST(Planning, DISABLED_path_target_random) {
             EXPECT_FALSE(true);
         }
 
-        // EXPECT_TRUE(checkPathContinuous(path, RobotConstraints{}));
+        // EXPECT_TRUE(checkTrajectoryContinuous(path, RobotConstraints{}));
     }
 
-    EXPECT_LT(failure_count, 100)
-        << "Failure rate should be less than 10 percent for path target";
+    EXPECT_LT(failure_count, 150)
+        << "Failure rate should be less than 15 percent for path target";
```

Comment:
Where does this number come from?

Reply:
From my ass.

These tests are garbage and heuristic, it's not really reasonable to put hard expectations on them.